src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -1639,6 +1662,59 @@ public void awaitingPickupRequestClosedDateIsNotPresentAfterStatusUpdateFromOpen
     assertThat(updatedRequest.getString("awaitingPickupRequestClosedDate"), is(nullValue()));
   }
 
+  @Test
+  public void canFindRequestsWithIsbnIdentifier()
+    throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+
+    final UUID nodRequestId = UUID.randomUUID();
+    final UUID smallAngryPlanetRequestId = UUID.randomUUID();
+    final UUID issnIdentifierId = UUID.randomUUID();
+    final UUID isbnIdentifierId = UUID.randomUUID();
+    final String isbn = "978-92-8011-566-10";
+
+    final ItemSummary nod = new ItemSummary("Nod", "565578437802")
+      .addIdentifier(issnIdentifierId, "978-92-8011-566-9")
+      .addIdentifier(UUID.randomUUID(), "2193988")
```

Comment:
Why are the random UUIDs inline on this line and a constant above?

Reply:
It is actually means 'an another identifier, in which we're not interesting in'.

Reply:
Maybe extract a constant to describe that intent.

I thought the test did search for that identifier?

Reply:
Ok, sure. No, the test do a search for ISBN identifier.